package.json

```diff
@@ -47,6 +47,8 @@
     "copy-dir": "^0.3.0",
     "cron": "1.7.2",
     "dockerode": "^2.5.8",
+    "dotenv": "^8.2.0",
```

Comment:
need to revert this file
dotenv is dev dependency 
and downloadjs is not in use